src/object/srv_ec_aggregate.c

```diff
@@ -1323,14 +1324,14 @@ agg_peer_update_ult(void *arg)
 out_bulk:
 	if (stripe_ud->asu_write_par)
 		crt_bulk_free(ec_agg_in->ea_bulk);
-out_rpc:
+out_agg:
 	if (ec_agg_in->ea_remove_nr) {
 		D_FREE(ec_agg_in->ea_remove_recxs.ca_arrays);
 		D_FREE(ec_agg_in->ea_remove_eps.ca_arrays);
 	}
+out_rpc:
```

Comment:
also no need to check rpc != NULL if we get here assuming we change back to out: above